.github/github_workflow_scripts/utils.py

```diff
@@ -248,3 +251,48 @@ def get_doc_reviewer(content_roles: dict[str, Any]) -> str:
     if not (reviewer := content_roles.get(DOC_REVIEWER_KEY)):
         raise ValueError("Cannot get doc reviewer")
     return reviewer
+
+
+def get_content_roles_from_blob() -> dict[str, Any] | None:
+    """
+    Helper method to retrieve the 'content_roles.json` from
+    the `demisto/content` master blob.
+
+    Returns:
+    - `dict[str, Any]` representing the content roles. See `.github/content_roles.json` for
+    the expected structure. If there's any failure getting/reading the resource,
+    we return `None`.
+    """
+
+    roles = None
+
+    try:
+        response = requests.get(CONTENT_ROLES_BLOB_MASTER_URL)
+        response.raise_for_status()  # Raise an error for bad status codes
+        json_data = response.json()
+        print(f"Successfully retrieved {CONTENT_ROLES_FILENAME} from blob")
+        roles = json_data
```

Comment:
```suggestion
        roles = response.json()
        print(f"Successfully retrieved {CONTENT_ROLES_FILENAME} from blob")
```

Reply:
https://github.com/demisto/content/pull/34932/commits/f4295974f2d6977e2b0aefd4fc2681002a6e00bd

Reply:
Here I meant 2 things:

1. Write the success message only after you parse it to json.
2. Save the results directly to `roles` instead of saving to a temporary unused variable.

Reply:
Dealt with 1. in https://github.com/demisto/content/pull/34932/commits/f4295974f2d6977e2b0aefd4fc2681002a6e00bd and
2. in https://github.com/demisto/content/pull/34932/commits/ffe733f5e184b1b6c48bce39cc070868c9917fe9